src/app/personside/infotabs/meldinger/traadvisning/verktoylinje/journalforing/VelgSak.tsx

```diff
@@ -97,13 +92,13 @@ export function sakKategori(sak: JournalforingsSak): SakKategori {
 function VelgSak(props: Props) {
     const fnr = useSelector(fnrSelector);
     const valgtKategori = useFieldState(SakKategori.FAG);
-    const gsakSaker: FetchResult<Array<JournalforingsSak>> = JournalforingUtils.useSammensatteSaker(fnr);
-    const psakSaker: FetchResult<Array<JournalforingsSak>> = JournalforingUtils.usePensjonSaker(fnr);
+    const result: FetchResult<Result> = JournalforingUtils.useSammensatteSaker(fnr);
```

Comment:
Siden vi har gått fra to til ett api-kall, så tror jeg vi kan strømlinjeforme denne komponenten og fjerne noe av den defansive stilen som vi brukte her før.

Typisk når vi bare henter fra ett endepunkt kan vi gjøre;
```
const result: FetchResult<Result> = JournalforingUtils.useSammensatteSaker(fnr);
if (isPending(result)) {
  return <Spinner type="XL" />;
} else if (hasError(result)) {
  return (
    <AlertStripeFeil className="blokk-xxxs">
      Feilet ved uthenting av saker
    </AlertStripeAdvarsel>
  );
}

const { saker, feiledeSystemer }  = result.data;
const fordelteSaker = fordelSaker(saker);

const feiledeSystemerAlerts = feiledeSystemer.map((system) => (
  <AlertStripeAdvarsel className="blokk-xxxs" key={system}>
    Feil ved uthenting av saker fra saker fra {system}
  </AlertStripeAdvarsel>
));
```

Når vi lener oss på typesystemet på denne måten så tvinges vi til å håndtere feil-situasjoner, ved at man f.eks ikke får tilgang til `data` før man har sjekket at kallet ikke feilet. 

Per i dag så vil komponenten havne i en nokså rar state om selve rest-kallet fra `useSammensatteSaker` feiler, siden da vil `hasData` være `false`, og både `saker` og `errors` vil bestå av tomme lister.